content/docs/replication/deployment/installation.md

```diff
@@ -27,38 +27,7 @@ for future use during management operations.
 Please follow the steps [here](../install-repctl) to install & configure Dell Replication Controller using repctl.
 
 #### Using the installation script
-Repeat the following steps on all clusters where you want to configure replication:
-
-```shell
-git clone github.com/dell/csm-replication
-cd csm-replication
-kubectl create ns dell-replication-controller
-# Copy and  modify values.yaml file if you wish to customize your deployment in any way
-cp ./helm/csm-replication/values.yaml ./myvalues.yaml
-bash scripts/install.sh --values ./myvalues.yaml
-```
->Note: Current installation method allows you to specify custom `<FQDN>:<IP>` entries to be appended to controller's `/etc/hosts` file. It can be useful if controller is being deployed in private environment where DNS is not set up properly, but kubernetes clusters use FQDN as API server's address.
-> The feature can be enabled by modifying `values.yaml`.
->``` hostAliases:
-> - ip: "10.10.10.10"
->   hostnames: 
->     - "foo.bar"
-> - ip: "10.10.10.11"
->   hostnames: 
->     - "foo.baz"
-
-This script will do the following:
-1. Install `DellCSIReplicationGroup` CRD in your cluster
-2. Install `dell-replication-controller`
-
-
-After the installation ConfigMap will consist of only the `logLevel` field, to add the rest configuration to the cluster do the following:
-* Update the configuration in `deploy/config.yaml` after going through the guide [here](../configmap-secrets)
-* Run the following commands to update and complete the installation
-  ```shell
-  cd csm-replication
-  kubectl create configmap dell-replication-controller-config --namespace dell-replication-controller --from-file deploy/config.yaml -o yaml --dry-run | kubectl apply -f -
-  ```
+Please follow the steps [here](../install-script) to install & configure Dell Replication Controller using script.
```

Comment:
Did rebase impact this?

Reply:
Yes. We had eliminated the inline instructions and put them into their own page. Rebase reverted this change. @santhoshatdell please confirm.

Reply:
Thought so... I saw the other page in main...